rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy

```diff
@@ -1451,8 +1462,8 @@ class ProjectController extends ControllerBase{
                     format: respFormat
             ])
         }
-        def properties = frameworkService.loadProjectProperties(project)
-        def resultValue= properties.get(key_)
+        def resultValue= project.getProperty(key_)
+        checkScheduleChanges(project, propValueBefore?new Properties([(key_): propValueBefore]):new Properties([(key_): '']), new Properties([(key_): resultValue]))
```

Comment:
try to split this line because it is difficult to understand to other developers

Reply:
:ok_hand: